prosody.rb

```diff
@@ -2,12 +2,9 @@
 
 class Prosody < Formula
   homepage "http://prosody.im"
-  #url "https://prosody.im/downloads/source/prosody-0.9.6.tar.gz"
-  #sha1 "ed8fd64cbe7ad9f33c451535258cd5fa2de0834e"
-  #version "0.9.6"
-  url "https://prosody.im/tmp/0.9.7/prosody-0.9.7.tar.gz"
-  sha1 "5515e15077ea0e8e26b83614a02ad632374a256b"
-  version "0.9.7"
+  url "https://prosody.im/tmp/0.9.7/prosody-0.9.8.tar.gz"
```

Comment:
In fact, please change it to "https://prosody.im/downloads/source/prosody-0.9.8.tar.gz", we shouldn't use /tmp/ files, and it was used mainly because 0.9.7 wasn't released at the time, but had a fix required for homebrew.